lib/matplotlib/axis.py

```diff
@@ -604,6 +604,14 @@ class Ticker(object):
     locator = None
     formatter = None
 
+    def __init__(self, ticker=None):
```

Comment:
I have a slight preference to stay with the pattern the `Artists` use of an `update_from` method, but not enough to hold up the PR over.